### ISSUE TITLE
eager func pattern需要支持捕获literal value

### ISSUE THREAD

**Issue by yfchen-byted** [BD-Seed-HHW/xpu_graph#350]
### Describe the feature🚀

以layernorm为例：

```python
def layer_norm(x, gamma, beta, eps):
  mean = torch.mean(x, [-1],keepdim=True)
  var = torch.var(x, [-1], keepdim=True, correction=True)
  norm = (x - mean) / (var + eps)
  return x * gamma + beta
```

在eager pattern中，
1. 因为通过dispatch生成 pattern graph的过程总是会将eps转换为具体的浮点数，无法将其与pattern graph的placeholder关联，
2. 导致eager pattern无法捕获到原图中的eps
3. 结果是，匹配、替换后的eps变成了replace pattern中提供的eps，而非图中原始的eps。

需要修改dispatch出的pattern graph，使得pattern能够捕获到原图中的eps

### References

N/A

### Environment details

N/A
